lib/logproto/tests/test-server-options.c

```diff
@@ -40,7 +40,6 @@ Test(log_proto, test_log_proto_server_options_limits)
                "LogProtoServerOptions.init_buffer_size is not initialized properly, init_buffer_size=%d", opts.init_buffer_size);
   cr_assert_gt(opts.max_buffer_size, 0,
                "LogProtoServerOptions.max_buffer_size is not initialized properly, max_buffer_size=%d", opts.max_buffer_size);
-  log_proto_server_options_destroy(&opts);
```

Comment:
`opts` is local, this is not freed by the `teardown()` function, so the `destroy()` call should not be removed.

This same is true for the other 2 occurrences.